ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/StyleFactory.kt

```diff
@@ -400,6 +405,7 @@ internal class StyleFactory(
             is TabControlToggleComponent -> createTabControlToggleComponentStyle(component)
             is TabControlComponent -> tabControl.errorIfNull(nonEmptyListOf(PaywallValidationError.TabControlNotInTab))
             is TabsComponent -> createTabsComponentStyle(component)
+            is VideoComponent -> createVideoComponentStyle(component)
```

Comment:
[Not related to this line!]

We might need to handle "header video" components, same way we do for images: 
https://github.com/RevenueCat/purchases-android/blob/9ef023d18085f3083a584e2627ba670f1d4a0711/ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/StyleFactory.kt#L162

This way, it would extend beyond safe areas.

Reply:
I actually had some of it done before I got scrambled

Reply:
I'm increasing my logcat buffer and trying to track that down now